.github/workflows/lte-integ-test-magma-deb.yml

```diff
@@ -9,6 +9,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# owner: @magma/approvers-gw
+# purpose: Run integration tests vs an environment based on a debian AGW artifact build with Make
```

Comment:
How about `Run LTE integration tests against the AGW services from the Make-built Debian package` ?